cotk/metric/bleu.py

```diff
@@ -195,6 +195,9 @@ def close(self):
 			raise RuntimeError("The metric has not been forwarded data correctly.")
 		if len(self.hyps) == 1:
 			raise RuntimeError("Self-Bleu can't be computed because there is only 1 generated sentence.")
+		if self.sample <= 1:
+			raise RuntimeError('`self.sample` should be more than 1, \

```

Comment:
Pay attention to end of line. This is a windows endline CRLF,  change to Unix endline LF.